tests/v1.0/fail/extends-invalid-type.yaml

```diff
@@ -0,0 +1,6 @@
+overlay: 1.0.0
+info:
+  title: Invalid `extends` type
+  version: 1.0.0
+extends: {}
+actions: []
```

Comment:
Better add a valid action here to make sure that the `overlay` field causes the fail